BridgeCareApp/BridgeCare/DataAccessLayer/SummaryReport/BridgeData.cs

```diff
@@ -0,0 +1,175 @@
+            try
+            {   
+                var connection = new SqlConnection(dbContext.Database.Connection.ConnectionString);
+                using (var cmd = new SqlCommand(selectSimulationStatement, connection))
+                {
+                    cmd.CommandTimeout = 180;
+                    var dataAdapter = new SqlDataAdapter(cmd);
+                    dataAdapter.Fill(simulationDataTable);
+                    dataAdapter.Dispose();
+                }
+            }
+            catch (SqlException ex)
+            {
+                HandleException.SqlError(ex, "Simulation_" + simulationModel.NetworkId + "_" + simulationModel.SimulationId);
+            }
+            catch (OutOfMemoryException ex)
+            {
+                HandleException.OutOfMemoryError(ex);
+            }
+
+            return simulationDataTable;
+        }
+
+        /// <summary>
+        /// Get Project, Cost related data from dynamic table Report_x_y, x = Network Id, y = Simulation Id
+        /// </summary>
+        /// <param name="simulationModel"></param>
+        /// <param name="dbContext"></param>
+        /// <param name="simulationYears"></param>
+        /// <returns></returns>
+        public IQueryable<ReportProjectCost> GetReportData(SimulationModel simulationModel, BridgeCareContext dbContext, List<int> simulationYears)
+        {            
+            IQueryable<ReportProjectCost> rawQueryForReportData = null;
+            var years = string.Join(",", simulationYears);
+            var selectReportStatement = "SELECT SECTIONID, TREATMENT, COST_, YEARS " + " FROM REPORT_" + simulationModel.NetworkId + "_" + simulationModel.SimulationId + " WITH(NOLOCK) WHERE BUDGET = 'actual_spent' AND YEARS IN (" + years + ")";
+            try
+            {
+                rawQueryForReportData = dbContext.Database.SqlQuery<ReportProjectCost>(selectReportStatement).AsQueryable();
+            }
+            catch (SqlException ex)
+            {
+                HandleException.SqlError(ex, "Report_" + simulationModel.NetworkId + "_" + simulationModel.SimulationId);
+            }
+            catch (OutOfMemoryException ex)
+            {
+                HandleException.OutOfMemoryError(ex);
+            }
+
+            return rawQueryForReportData;
+        }
+
+        #region private methods
+        private static string GetDynamicColumns(List<int> simulationYears, string dynamicColumns)
+        {
+            foreach (var year in simulationYears)
+            {
+                dynamicColumns = dynamicColumns + ", DECK_SEEDED_" + year + ", SUP_SEEDED_" + year + ", SUB_SEEDED_" + year + ", CULV_SEEDED_" + year + ", DECK_DURATION_N_" + year + ", SUP_DURATION_N_" + year + ", SUB_DURATION_N_" + year + ", CULV_DURATION_N_" + year;
```

Comment:
Depending on whether these column names are used elsewhere, consider moving them to const strings.

Reply:
Good point, did it, partial string were common. Thanks.